content/docs/lifting-state-up.md

```diff
@@ -232,11 +232,11 @@ class TemperatureInput extends React.Component {
 }
 ```
 
-Now let's turn to the `Calculator` component.
+अब `Calculator` कौम्पोनॅन्ट की ओर मुड़ते हैं।
 
-We will store the current input's `temperature` and `scale` in its local state. This is the state we "lifted up" from the inputs, and it will serve as the "source of truth" for both of them. It is the minimal representation of all the data we need to know in order to render both inputs.
+हम वर्तमान इनपुट के `temperature` और `scale` को उसके लोकल state में जमा करेंगे. यह वही state है जिसे हमने इनपुट्स से "ऊपर उठा लिया", और यह उन दोनों के लिए "सत्य का एक स्रोत" के रूप में काम करेगी| यह दोनों इनपुट्स को रेंडर करने के लिए हमारे द्वारा आवश्यक सभी डेटा का न्यूनतम प्रतिनिधित्व है।
```

Comment:
`में जमा करेंगे.` => `में जमा करेंगे ।`

Reply:
`|` => `।`

let's change this everywhere in the whole file

Reply:
👍